package.json

```diff
@@ -31,14 +31,16 @@
     "strip-ansi-cli": "^2.0.0"
   },
   "dependencies": {
+    "ascii-table": "0.0.9",
+    "bn.js": "^5.1.1",
     "bs58": "^4.0.1",
     "chalk": "^4.0.0",
     "flagged-respawn": "^1.0.1",
     "is-ci": "^2.0.0",
     "jest-environment-node": "^26.0.0",
     "mixpanel": "^0.11.0",
     "ncp": "^2.0.0",
-    "near-api-js": "^0.23.1",
+    "near-api-js": "git+https://github.com/near/near-api-js.git",
```

Comment:
@ilblackdragon needs fix. When you test locally use https://classic.yarnpkg.com/en/docs/cli/link/

Reply:
Yep, waiting for the new release.